src/peergos/server/net/MutationHandler.java

```diff
@@ -0,0 +1,73 @@
+package peergos.server.net;
+
+import com.sun.net.httpserver.*;
+import peergos.server.*;
+import peergos.server.mutable.*;
+import peergos.server.util.*;
+import peergos.shared.crypto.hash.*;
+import peergos.shared.mutable.*;
+import peergos.shared.util.*;
+
+import java.io.*;
+import java.util.*;
+import java.util.logging.*;
+
+public class MutationHandler implements HttpHandler {
```

Comment:
class docstring plz.

Reply:
what is this used by?

Reply:
This is used by the peergos server (UserService)